yt_dlp/extractor/tver.py

```diff
@@ -39,6 +51,10 @@ def _real_initialize(self):
 
     def _real_extract(self, url):
         path, video_id = self._match_valid_url(url).groups()
+        if path[0:2] == "lp":
```

Comment:
y not just `if path == 'lp'` ?

Reply:
Line 53 works like `path, video_id = ('lp / c0000239','c0000239')`, so `if path =='lp'` does not work as expected.
But it's confusing so I'll fix it.

Reply:
it was changed in https://github.com/yt-dlp/yt-dlp/pull/3075/commits/ba37011b37948e01941593fd5514b2f779ae4d3f